src/sunrise/core.cljs

```diff
@@ -1,85 +1,13 @@
 (ns sunrise.core
-  (:require [reagent.core :as r]))
+  (:require [reagent.core :as reagent]
+            [sunrise.component :as component]))
 
 (defn get-window-dimensions []
   (let [width (.-innerWidth js/window)
         height (.-innerHeight js/window)]
     {:width width :height height}))
 
-(def display-state (r/atom {:window (get-window-dimensions)}))
+(def display-state (reagent/atom {:window (get-window-dimensions)}))
```

Comment:
Good idea to move away from single character names

Reply:
Glad we agree!